ffi/src/lib.rs

```diff
@@ -52,6 +52,17 @@ pub mod transaction;
 
 pub(crate) type NullableCvoid = Option<NonNull<c_void>>;
 
+#[derive(Default)]
```

Comment:
I think you're not using `CStringMap` in this PR? Can we revert this change if it's not needed?